src/components/features/RenderPostsOfSelctCategory/RenderPostsOfSelectCategories.js

```diff
@@ -0,0 +1,42 @@
+import { useParams } from 'react-router';
+import { useSelector } from 'react-redux';
+import { getPostsByCategory } from '../../../redux/postsRedux';
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+import { dateToStr } from '../../../utils/dateToStr';
+import { Link } from 'react-router-dom';
+
+
+
+
+const RenderPostsofSelectCategories = () => {
```

Comment:
Tutaj mam trochę problem z tym. Zasadniczo nie spotkałem się z konwencją nazywania komponentów jako czasowniki. Komponenty zazwyczaj są rzeczownikami. Tak samo klasy. Z kolei czasowniki dotyczą metod. 
Czyli raczej skłaniałbym się do nazwy `PostsOfSelectedCategories` (w czasie present simple). Gdyby komponent/klasa z kolei miała jakąś metodę, to wtedy nazwa `renderPostOfSelectedCategories` miałaby sens. 

Do poczytania: https://medium.com/@wittydeveloper/react-components-naming-convention-%EF%B8%8F-b50303551505

Dotyczy wszystkich komponentów nazwanych czasownikiem, ale nie będę Ci tego blokował z tego powodu.

Reply:
Rozumiem, dzięki za uwagę